src/config.rs

```diff
@@ -141,6 +149,7 @@ impl Config {
                 "TOKIO_CONSOLE_LISTEN_ADDRESS",
                 "127.0.0.1:6669".parse().unwrap(),
             )?,
+            metrics_refresh_enabled: var_optional("ENABLE_METRICS_REFRESH", true)?,
```

Comment:
Same comment here as previous discussion of `enable_metrics_refresh` vs `metrics_refresh_enabled` but for the env var names (`TOKIO_CONSOLE_ENABLED` vs `ENABLE_METRICS_REFRESH`)

Reply:
Ah oops, thanks for catching that.